services/mongodbflex/model_dev_azure_com_schwarzit_schwarzit_stackit_mongodb_stackit_mongodb_api_git_handlers_infra_flavor.go

```diff
@@ -1,19 +0,0 @@
-/*
-STACKIT MongoDB Service API
```

Comment:
Its an unused component, but since it is being removed, it should go in the changelog and make a new release

Reply:
As discussed, this is not used anywhere and all these unused components will be removed soon from the generated OASs